WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Posts/DashboardPostsCardCell.swift

```diff
@@ -14,13 +14,17 @@ import UIKit
 /// Eg.: The API might return that the user has no posts at all but
 /// they actually have a draft saved locally. In this case we show the drafts.
 class DashboardPostsCardCell: UICollectionViewCell, Reusable {
-    private var cardFrameView: BlogDashboardCardFrameView?
+    /// THe first card frame view on this cell
+    private var firstCardFrameView: BlogDashboardCardFrameView?
```

Comment:
Do we still need this private var? Can we delete this since now the `cardFrameView` is being passed as an argument in the `PostsCardViewControllerDelegate` methods?

Slack ref: p1646221454174039/1646188721.858469-slack-C0290FLA0RM

Reply:
We do! :(

It's used by the dequeuing logic. I can probably refactor that to have the VC reference instead but given that the changes on this PR are already extensive I prefer to not include more stuff. I added as a "nice to have" on our board.